internal/pkg/cli/pipeline_init_test.go

```diff
@@ -232,54 +200,7 @@ func TestInitPipelineOpts_Ask(t *testing.T) {
 			},
 			mockSessProvider: func(m *mocks.MocksessionProvider) {},
 
-			expectedRepoURL:           githubAnotherURL,
-			expectedGitHubOwner:       githubOwner,
-			expectedRepoName:          githubAnotherRepoName,
-			expectedGitHubAccessToken: githubToken,
-			expectedRepoBranch:        "main",
-			expectedEnvironments:      []string{"test", "prod"},
-			expectedEnvConfigs:        mockEnvConfigs,
-			expectedError:             nil,
-		},
-		"no flags, success case for CodeCommit": {
-			inEnvironments: []string{},
-			inRepoURL:      "",
-			buffer:         *bytes.NewBufferString("archer\tgit@github.com:goodGoose/bhaOS (fetch)\narcher\thttps://github.com/badGoose/chaOS (push)\narcher\thttps://git-codecommit.us-west-2.amazonaws.com/v1/repos/repo-man (fetch)\narcher\tssh://git-codecommit.us-west-2.amazonaws.com/v1/repos/repo-woman (push)\narcher\tcodecommit::us-west-2://repo-man (fetch)\n"),
-
-			mockSelector: func(m *mocks.MockpipelineEnvSelector) {
-				m.EXPECT().Environments(pipelineSelectEnvPrompt, gomock.Any(), "my-app", gomock.Any()).Return([]string{"test", "prod"}, nil)
-			},
-			mockStore: func(m *mocks.Mockstore) {
-				m.EXPECT().GetEnvironment("my-app", "test").Return(&config.Environment{
-					Name:   "test",
-					Region: "us-west-2",
-				}, nil)
-				m.EXPECT().GetEnvironment("my-app", "prod").Return(&config.Environment{
-					Name:   "prod",
-					Region: "us-west-2",
-				}, nil)
-			},
-			mockRunner: func(m *mocks.Mockrunner) {
-				m.EXPECT().Run(gomock.Any(), gomock.Any(), gomock.Any()).Return(nil)
-			},
-			mockPrompt: func(m *mocks.Mockprompter) {
-				m.EXPECT().SelectOne(pipelineSelectURLPrompt, gomock.Any(), gomock.Any(), gomock.Any()).Return(codecommitSSHURL, nil).Times(1)
-			},
-			mockSessProvider: func(m *mocks.MocksessionProvider) {
-				m.EXPECT().Default().Return(&session.Session{
-					Config: &aws.Config{
-						Region: aws.String("us-west-2"),
-					},
-				}, nil)
-			},
-
-			expectedRepoURL:          codecommitSSHURL,
-			expectedRepoName:         codecommitAnotherRepoName,
-			expectedRepoBranch:       "main",
-			expectedCodeCommitRegion: codecommitRegion,
-			expectedEnvironments:     []string{"test", "prod"},
-			expectedEnvConfigs:       mockEnvConfigs,
-			expectedError:            nil,
+			expectedError: nil,
```

Comment:
nit: I think `expectedError`is initialized as nil so declaring it here would be redundant.

Reply:
Done! deleted all `expectedError: nil`